datafusion/expr/src/logical_plan/ddl.rs

```diff
@@ -192,6 +192,8 @@ pub struct CreateExternalTable {
     pub order_exprs: Vec<Expr>,
     /// File compression type (GZIP, BZIP2, XZ, ZSTD)
     pub file_compression_type: CompressionTypeVariant,
+    /// Whether the table is an infinite streams
+    pub unbounded: bool,
```

Comment:
I think, `fn hash<H: Hasher>(&self, state: &mut H)` function should use `self.unbounded` during hashing also.

Reply:
Ah yes, I miss that

Reply:
Thank you